### PR TITLE
feat: session tracking and i/o on traces

### PR DESCRIPTION
**Pull request by marcklingen**

### CODE REVIEW

**Comment by marcklingen** [comment]
deployed migrations to staging env

File: prisma/schema.prisma [hunk 1]
```diff
@@ -147,24 +148,45 @@ enum MembershipRole {
     VIEWER
 }
 
-model Trace {
+model TraceSession {
```
Comment (maxdeichmann) [inline prisma/schema.prisma:151]: I assume you did not want to take `Session` as this was taken above already?

File: prisma/schema.prisma [hunk 2]
```diff
@@ -147,24 +148,45 @@ enum MembershipRole {
...
+    projectId  String        @map("project_id")
+    project    Project       @relation(fields: [projectId], references: [id], onDelete: Cascade)
+    public     Boolean       @default(false)
+    bookmarked Boolean       @default(false)
 
     scores Score[]
 
     @@unique([projectId, externalId])
     @@index([projectId])
+    @@index([sessionId])
```
Comment (maxdeichmann) [inline prisma/schema.prisma:189]: thanks!

File: prisma/schema.prisma (hunk 1 above)
Comment (marcklingen) [reply prisma/schema.prisma:151]: yep, resolve clash with auth session, on all apis/sdks it will only be `session`

**Pull request merged**